### PR TITLE
Closes #1222: Update min-height of red bar to accommodate different theme settings.

### LINKED ISSUE #1222: Mobile menu/search break if you turn off institutional header wordmark logo
## Problem/Motivation
In working on getting Arizona.edu into Quickstart 2, we are noticing that when you turn off the institutional header wordmark logo in the theme settings (necessary for WWW) the red header area on small screens is broken.

image

### Describe the bug
When you turn the setting off, the redbar disappears on mobile. The search icons and menu hamburger still appear, but are not functional at all. The Mobile menu can not be navigated.

image

### To Reproduce

1. Go to '/admin/appearance/settings/az_barrio'
2. Uncheck the checkbox under Institutional Logo Settings
3. Go to the homepage and view on small screen
4. Attempt to access mobile menu

### Expected behavior

It is expected that the redbar would appear and the search/menu icons would still function. Most sites use the institutional wordmark, but it is redundant on Arizona.edu and some other sites.

### PR DESCRIPTION
**Pull request by danahertzberg**
## Description

Ensures mobile buttons are clickable regardless of wordmark appearance setting being enabled

## Related Issue

Closes #1222 

## How Has This Been Tested?

Probo

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I've added this Pull Request to the AZ Quickstart Github project and set it to "Review in progress".
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: themes/custom/az_barrio/css/uaqs_navigation_overlay_menu_scroll.css [hunk 1]
```diff
@@ -1,3 +1,8 @@
+/* For structuring header utilities*/
+header#header_arizona {
+  min-height: 50px;
+}
+
```
Comment (joeparsons) [inline themes/custom/az_barrio/css/uaqs_navigation_overlay_menu_scroll.css:5]: Looks like this `uaqs_navigation_overlay_menu_scroll.css` file isn't associated with any of AZ Barrio's [libraries](https://github.com/az-digital/az_quickstart/blob/main/themes/custom/az_barrio/az_barrio.libraries.yml) and isn't really used anywhere...

Comment (joeparsons) [inline themes/custom/az_barrio/css/uaqs_navigation_overlay_menu_scroll.css:5]:
```suggestion
```

Comment (danahertzberg) [reply themes/custom/az_barrio/css/uaqs_navigation_overlay_menu_scroll.css:5]: Could this be related to some previous/current issues of Quickstart not using Bootstrap styles? Let's do the 2 ticket solution like we discussed this morning for this one.

File: themes/custom/az_barrio/az_barrio.libraries.yml [hunk 2]
```diff
@@ -8,6 +8,9 @@ az-media-display:
     component:
       css/az-media-display.css: {}
 az-barrio-off-canvas-nav:
+  css:
+    component:
```
Comment (joeparsons) [inline themes/custom/az_barrio/az_barrio.libraries.yml:12]:
```suggestion
    theme:
```

**Comment by trackleft** [comment]
Sorry, it seems as though I left a lot of garbage in there.

**Comment by joeparsons** [comment]
I think we'll want a follow-up issue to look into separating `arizona-bootstrap` into 2 different libraries (one for CSS and one for JS) so we can attach the Bootstrap JS only where needed.

**Review by joeparsons — APPROVED** [review]

**Review by danahertzberg — COMMENTED** [review]
Tested all combinations of theme settings and it's working like a champ! Thank you @joeparsons!!

**Comment by joeparsons** [comment]
We may also want a follow-up to investigate whether it would be better for these styles to be added to Arizona Bootstrap instead of in the Drupal theme...

**Review by camikazegreen — APPROVED** [review]
Confirming that this will add a 50px red bar to sites that have the institutional wordmark disabled, but have the off-canvas mobile nave enabled. 
Individual follow up with the site owner will be required for isl.arizona.edu when we roll out this update.

**Review by trackleft — APPROVED** [review]

**Pull request merged**